website/data/docs-nav-data.json

```diff
@@ -154,10 +154,6 @@
               {
                 "title": "Contextual Functions",
                 "routes": [
-                  {
```

Comment:
These sidebar items currently lead to empty pages. We want to remove the empty pages, so we need to remove these sidebar items as well.